### PR TITLE
Clean up kubectl install page

### PR DESCRIPTION
**Pull request by cody-clark**
Fixes issue #13577

### CODE REVIEW

**Comment by cody-clark** [comment]
Preview: https://deploy-preview-13578--kubernetes-io-master-staging.netlify.com/docs/tasks/tools/install-kubectl/#additional-install-methods

**Comment by cody-clark** [comment]
Hey, Jared and Zach, this pass was just for reorganization.  All feedback welcome! 

assign/ @jaredbhatti 
assign/ @zacharysarah

**Comment by cody-clark** [comment]
Trying to assign again

/assign @jaredbhatti 
/assign @zacharysarah

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 1]
```diff
@@ -22,11 +22,82 @@ You must use a kubectl version that is within one minor version difference of yo
 
 {{% capture steps %}}
 
-## Install kubectl
+## Install kubectl binary using curl
```
Comment (zacharysarah) [inline content/en/docs/tasks/tools/install-kubectl.md]: s/using/with

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 2]
```diff
@@ -22,11 +22,82 @@ You must use a kubectl version that is within one minor version difference of yo
...
+    ```
 
-## Install kubectl binary using native package management
+    To find out the latest stable version (for example, for scripting), take a look at [https://storage.googleapis.com/kubernetes-release/release/stable.txt](https://storage.googleapis.com/kubernetes-release/release/stable.txt).
+
+2. Add the binary in to your PATH.
+{{% /tab %}}
+{{< /tabs >}}
+
+## Install kubectl on Linux distros
```
Comment (zacharysarah) [inline content/en/docs/tasks/tools/install-kubectl.md]: > Install kubectl on Linux

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 3]
```diff
@@ -22,11 +22,82 @@ You must use a kubectl version that is within one minor version difference of yo
```
Comment (jaredbhatti) [inline content/en/docs/tasks/tools/install-kubectl.md]: Overview section: Suggest revising the first sentence to: The Kubernetes command-line tool, [kubectl](/docs/user-guide/kubectl/), allows you to deploy and manage applications on Kubernetes.

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 4]
```diff
@@ -50,7 +121,7 @@ yum install -y kubectl
 {{< /tabs >}}
 
 
-## Install with snap on Ubuntu
+### Install with snap on Ubuntu
```
Comment (jaredbhatti) [inline content/en/docs/tasks/tools/install-kubectl.md]: Suggest change to "Install with snap". Most Linux distros support snapd.

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 5]
```diff
@@ -122,7 +197,7 @@ If you are on Windows and using [Powershell Gallery](https://www.powershellgalle
 
     {{< note >}}Updating the installation is performed by rerunning the two commands listed in step 1.{{< /note >}}
 
-## Install on Windows using Chocolatey or scoop
+### Install on Windows using Chocolatey or scoop
 
 To install kubectl on Windows you can use either [Chocolatey](https://chocolatey.org) package manager or [scoop](https://scoop.sh) command-line installer.
```
Comment (jaredbhatti) [inline content/en/docs/tasks/tools/install-kubectl.md]: "scoop" -> "Scoop"

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 6]
```diff
@@ -185,87 +262,12 @@ You can install kubectl as part of the Google Cloud SDK.
...
-
-2. Add the binary in to your PATH.
-{{% /tab %}}
-{{< /tabs >}}
-
-
-
 ## Configure kubectl
 
 In order for kubectl to find and access a Kubernetes cluster, it needs a [kubeconfig file](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/), which is created automatically when you create a cluster using kube-up.sh or successfully deploy a Minikube cluster. See the [getting started guides](/docs/setup/) for more about creating clusters. If you need access to a cluster you didn't create, see the [Sharing Cluster Access document](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
```
Comment (jaredbhatti) [inline content/en/docs/tasks/tools/install-kubectl.md]: Suggest moving the following link to the "What's Next" section: 
- See the [getting started guides](/docs/setup/) for more about creating clusters. 
- If you need access to a cluster you didn't create, see the [Sharing Cluster Access document](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 7]
```diff
@@ -185,87 +262,12 @@ You can install kubectl as part of the Google Cloud SDK.
...
-
-    To find out the latest stable version (for example, for scripting), take a look at [https://storage.googleapis.com/kubernetes-release/release/stable.txt](https://storage.googleapis.com/kubernetes-release/release/stable.txt).
-
-2. Add the binary in to your PATH.
-{{% /tab %}}
-{{< /tabs >}}
-
-
-
 ## Configure kubectl
```
Comment (jaredbhatti) [inline content/en/docs/tasks/tools/install-kubectl.md]: Suggest merging this section (Configure kubectl) with the one below (Check the kubectl configuration), and renaming the entire section "Verifying kubectl configuration (kubeconfig)".

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 8]
```diff
@@ -12,7 +12,7 @@ card:
 ---
 
 {{% capture overview %}}
-Use the Kubernetes command-line tool, [kubectl](/docs/user-guide/kubectl/), to deploy and manage applications on Kubernetes. Using kubectl, you can inspect cluster resources; create, delete, and update components; look at your new cluster; and bring up example apps.
+The Kubernetes command-line tool, [kubectl](/docs/user-guide/kubectl/), allows you to deploy and manage applications on Kubernetes. Using kubectl, you can inspect cluster resources; create, delete, and update components; look at your new cluster; and bring up example apps.
```
Comment (jaredbhatti) [inline content/en/docs/tasks/tools/install-kubectl.md]: The section sentence "Using kubectl, you can inspect cluster resources; create, delete, and update components; look at your new cluster; and bring up example apps." can be rewritten a bit better.

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 9]
```diff
@@ -179,93 +254,16 @@ You can install kubectl as part of the Google Cloud SDK.
...
-
-3. Move the binary in to your PATH.
-
-    ```
-    sudo mv ./kubectl /usr/local/bin/kubectl
-    ```
-{{% /tab %}}
-{{% tab name="Windows" %}}
-1. Download the latest release {{< param "fullversion" >}} from [this link](https://storage.googleapis.com/kubernetes-release/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe).
+In order for kubectl to find and access a Kubernetes cluster, it needs a [kubeconfig file](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/), which is created automatically when you create a cluster using kube-up.sh or successfully deploy a Minikube cluster. By default, kubectl configuration is located at `~/.kube/config`.
```
Comment (jaredbhatti) [inline content/en/docs/tasks/tools/install-kubectl.md]: I think kube-up.sh should be in code block format?

File: content/en/docs/tasks/tools/install-kubectl.md [hunk 10]
```diff
@@ -425,6 +425,8 @@ compinit
 {{% /capture %}}
 
 {{% capture whatsnext %}}
-[Learn how to launch and expose your application.](/docs/tasks/access-application-cluster/service-access-application-cluster/)
+* See the [getting started guides](/docs/setup/) for more about creating clusters. 
+* [Learn how to launch and expose your application.](/docs/tasks/access-application-cluster/service-access-application-cluster/)
```
Comment (jaredbhatti) [inline content/en/docs/tasks/tools/install-kubectl.md]: Include a link to Minikube installation

**Comment by tengqm** [comment]
@cody-clark new comments to address, :)

**Comment by cody-clark** [comment]
Sorry about that, @tengqm! I've addressed all of the comments now :)

**Pull request merged**